modules/rest/src/integrationTest/java/org/apache/ignite/internal/rest/authentication/ItAuthenticationTest.java

```diff
@@ -62,9 +62,7 @@ public class ItAuthenticationTest extends ClusterPerTestIntegrationTest {
 
     @BeforeEach
```

Comment:
No, it's here on purpose in order to *not* start a cluster at the beginning

Reply:
ok, I understand what you mean now. Let's leave it as-is, because this is the only comment and I don't want to wait for the TC again just because of this